pajbot/apiwrappers/twitch/helix.py

```diff
@@ -142,8 +142,7 @@ def request(self, method, endpoint, params, headers, authorization=None, json=No
                 time_to_wait = rate_limit_reset - utils.now()
                 time.sleep(math.ceil(time_to_wait.total_seconds()))
                 return super().request(method, endpoint, params, headers, authorization, json)
-            else:
-                raise e
+            raise e
```

Comment:
Let's add a little space
```suggestion

            raise e
```